section4/exercises/burrito.rb

```diff
@@ -1,19 +1,38 @@
-# Add the following methods to this burrito class and 
+# Add the following methods to this burrito class and
 # call the methods below the class:
 # 1. add_topping
 # 2. remove_topping
 # 3. change_protein
 
 class Burrito
   attr_reader :protein, :base, :toppings
+
   def initialize(protein, base, toppings)
     @protein  = protein
     @base     = base
     @toppings = toppings
   end
+
+  def add_topping(new_topping)
+      toppings.push(new_topping)
+      puts "Now my burrito has #{toppings}"
+  end
+
+  def remove_topping(topping)
+    puts "My burrito had #{toppings}... "
+    toppings.delete(topping)
+    puts "But I took out #{topping}. Now my burrito only has #{toppings}."
+  end
+
+  def change_protein(protein)
+    puts "Can I actually get #{protein} in my burrito instead?"
```

Comment:
Great work! Does this actually change the protein though? Like, if I called `p dinner.protein` AFTER `p dinner.change_protein('chicken')`, would it print out "Beans" or "Chicken"?